docs/api-reference/carto/overview.md

```diff
@@ -36,96 +36,78 @@ fetchMap({cartoMapId}).then(map => new Deck(map));
+```jsx
 import DeckGL from '@deck.gl/react';
-import {CartoLayer, setDefaultCredentials, API_VERSIONS, MAP_TYPES} from '@deck.gl/carto';
+import {VectorTileLayer, vectorQuerySource} from '@deck.gl/carto';
 
-setDefaultCredentials({accessToken: 'XXX'});
+function App() {
+  const data = vectorQuerySource({
+    accessToken: 'XXX',
+    connectionName: 'bigquery',
+    sqlQuery: 'SELECT * FROM cartobq.testtables.points_10k',
+  });
 
-function App({viewState}) {
-  const layer = new CartoLayer({
-    type: MAP_TYPES.QUERY,
-    connection: 'bigquery',
-    data: 'SELECT * FROM cartobq.testtables.points_10k',
+  const layer = new VectorTileLayer({
+    data,
     pointRadiusMinPixels: 2,
     getLineColor: [0, 0, 0, 200],
     getFillColor: [238, 77, 90],
     lineWidthMinPixels: 1
   });
 
-  return <DeckGL viewState={viewState} layers={[layer]} />;
+  return <DeckGL layers={[layer]} />;
 }
 ```
 
 ### Examples
 
-You can see working examples for the following:
+You can view a collection of working examples on the [CARTO documentation website](https://docs.carto.com/carto-for-developers/carto-for-deck.gl/examples).
+
+### CARTO Layers
 
-- [Scripting](https://carto.com/developers/deck-gl/examples/): Quick scripting examples to play with the module without NPM or Webpack. If you're not a web developer, this is probably what you're looking for.
+The CARTO module contains a number of custom layers which can be used to visualize the data, which work in conjunction with the [data source](#carto-data-sources) functions:
 
-- [React](https://github.com/CartoDB/viz-doc/tree/master/deck.gl/examples/react): Integrate in a React application.
+- [H3TileLayer](./h3-tile-layer.md)
+- [QuadbinTileLayer](./quadbin-tile-layer.md)
+- [RasterTileLayer](./raster-tile-layer.md)
+- [VectorTileLayer](./vector-tile-layer.md)
 
-- [Pure JS](https://github.com/CartoDB/viz-doc/tree/master/deck.gl/examples/pure-js): Integrate in a pure js application, using webpack.
+### CARTO Data sources
 
-### CARTO credentials
+There are a number of [data source functions](./data-sources.md) for accessing data from the CARTO platform. These provide a simple way to fetch data from the CARTO platform, which can then be used with deck.gl layers.
 
-This is an object to define the connection to CARTO, including the credentials (and optionally the parameters to point to specific API endpoints).
+#### Authentication
 
-These are the available properties:
+When defining a [data source](./data-sources.md) it is necessary to provide a:
 
-- `apiBaseUrl` (optional): base URL for requests to the API (can be obtained in the CARTO 3 Workspace). Default: `https://gcp-us-east1.api.carto.com`.
-- `accessToken` (optional): token to authenticate/authorize requests to the Maps API (private datasets)
-- `mapsUrl` (optional): Maps API URL Template. By default it is derived from `apiBaseUrl`:
-  - `https://{apiBaseUrl}/v3/maps`
+- `accessToken`: token to authenticate/authorize requests to the CARTO API,
```

Comment:
Link to https://docs.carto.com/carto-for-developers/key-concepts/authentication-methods